201-vmss-azureautomationdsc/azuredeploy.json

```diff
@@ -0,0 +1,739 @@
+                "South Central US",
+                "North Central US",
+                "North Europe",
+                "West Europe",
+                "Southeast Asia",
+                "East Asia",
+                "Japan West",
+                "Japan East",
+                "Brazil South",
+                "Australia East",
+                "Australia Southeast"
+            ],
+            "metadata": {
+                "description": "Azure region where all resources will be deployed. This is distinct from resource group location."
+            }
+        },
+        "vmSku": {
+            "type": "string",
+            "defaultValue": "Standard_A2",
+            "metadata": {
+                "description": "VM size for the VM Scale Set + the mgmt VM"
+            }
+        },
+        "windowsOSVersion": {
+            "type": "string",
+            "defaultValue": "2012-R2-Datacenter",
+            "metadata": {
+                "description": "The Windows Server version for the VM. This will pick a fully patched image of this given Windows version"
+            }
+        },
+        "vmssName": {
+            "type": "string",
+            "defaultValue": "ScaleSet",
+            "metadata": {
+                "description": "String used as a base for naming resources (9 characters or less). A hash is prepended to this string for some resources, and resource-specific information is appended."
+            },
+            "maxLength": 9
+        },
+        "instanceCount": {
+            "type": "int",
+            "defaultValue": 5,
+            "metadata": {
+                "description": "Number of VM instances (100 or less)."
+            },
+            "maxValue": 100
+        },
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "Admin username on all VMs."
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Admin password on all VMs."
+            }
+        },
+        "registrationKey": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Registration key to use to onboard to the Azure Automation DSC pull/reporting server"
+            }
+        },
+        "registrationUrl": {
+            "type": "string",
+            "metadata": {
+                "description": "Registration url of the Azure Automation DSC pull/reporting server"
+            }
+        },
+        "nodeConfigurationName": {
+            "type": "string",
+            "defaultValue": "webServer.localhost",
+            "metadata": {
+                "description": "The name of the node configuration, on the Azure Automation DSC pull server, that this node will be configured as"
+            }
+        },
+        "configurationMode": {
+            "type": "string",
+            "defaultValue": "ApplyAndAutoCorrect",
```

Comment:
This isn't the DSC default. Are users aware that it will always autocorrect via the description? They may be but just making sure